templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -150,6 +150,9 @@ if [ "$TEST" = "s3" ]; then
   sed -i -e '$a s3_test: true\
 minio_access_key: "'$MINIO_ACCESS_KEY'"\
 minio_secret_key: "'$MINIO_SECRET_KEY'"' vars/main.yaml
+
+  echo "PULP_API_ROOT=/rerouted/djnd/" >> "$GITHUB_ENV"
+  export PULP_API_ROOT="/rerouted/djnd/"
```

Comment:
This should really be the lift of that feature. Do we want to set the vars in a different way?

cc @fao89

Reply:
I think this is good